docs/install/install-client.rst

```diff
@@ -52,55 +52,31 @@ Other Linux Distributions
 
 .. warning::
 
-   These are guidelines for installing Flocker on a Linux distribution which we do not provide native packages for.
+   These are guidelines for installing Flocker on a Linux distribution for which we do not provide native packages.
    These guidelines may require some tweaks, depending on the details of the Linux distribution in use.
 
 Before you install ``flocker-cli`` you will need a compiler, Python 2.7, and the ``virtualenv`` Python utility installed.
 
-To install these with the ``yum`` package manager, run:
+To install these pre-requisites with the ``yum`` package manager, run:
 
-.. prompt:: bash alice@mercury:~$
-
-   sudo yum install gcc python python-devel python-virtualenv libffi-devel openssl-devel
-
-To install these with ``apt``, run:
-
-.. prompt:: bash alice@mercury:~$
-
-   sudo apt-get update
-   sudo apt-get install gcc libssl-dev libffi-dev python2.7 python-virtualenv python2.7-dev
-
-Then run the following script to install ``flocker-cli``:
-
-:version-download:`linux-install.sh.template`
-
-.. version-literalinclude:: linux-install.sh.template
-   :language: sh
-
-Save the script to a file and then run it:
-
-.. prompt:: bash alice@mercury:~$
+.. task:: cli_pip_prereqs centos-7
+   :prompt: alice@mercury:~$
 
-   sh linux-install.sh
 
-The ``flocker-deploy`` command line program will now be available in :file:`flocker-tutorial/bin/`:
+To install these pre-requisites with the ``apt`` package manager, run:
 
-.. version-code-block:: console
+.. task:: cli_pip_prereqs ubuntu-15.04
+   :prompt: alice@mercury:~$
 
-   alice@mercury:~$ cd flocker-tutorial
-   alice@mercury:~/flocker-tutorial$ bin/flocker-deploy --version
-   |latest-installable|
-   alice@mercury:~/flocker-tutorial$
+Then run the following commands to install ``flocker-cli`` in a Python virtualenv:
```

Comment:
This sentence should keep the same structure as the other instructions - 
Change
+Then run the following commands to install ``flocker-cli`` in a Python virtualenv:
to
+To install ``flocker-cli`` in a Python virtualenv, run: